ol-keycloak/oltheme/src/main/resources/theme/ol/login/template.ftl

```diff
@@ -81,7 +81,7 @@
       <header class="pf-v5-c-login__main-header">
         <div class="pf-v5-u-mb-2xl">
           <h1 class="pf-v5-c-title pf-m-4xl">
-            <a href="https://mit.edu" class="logo-link pf-v5-u-display-flex pf-v5-u-justify-content-center">
+            <a href="${open.homeUrl}" class="logo-link pf-v5-u-display-flex pf-v5-u-justify-content-center">
```

Comment:
Does this actually work?  I'm getting an error:
```
        - Failed at: ${open.homeUrl}  [in template "template.ftl" in macro "registrationLayout" at line 84, column 22]
        - Reached through: @layout.registrationLayout displayMes...  [in template "error.ftl" at line 2, column 1]
```

Reply:
This worked for me, that error would make me think the latest code wasn't loaded in, but you wouldn't be seeing any of the rest of it either if that was the case. That particular property isn't conditional like `loginAttempt` is...